ViteMaDose/ViewModels/CentresList/CentresListViewModel.swift

```diff
@@ -274,3 +301,26 @@ extension CentresListViewModel: CentresListViewModelProvider {
         return bookingUrl
     }
 }
+
+// MARK: - Sort centre
+
+extension CentresListViewModel {
+
+    func auPlusProche(_ centre1: VaccinationCentre, _ centre2: VaccinationCentre) -> Bool {
+        // TODO: Calculate by distance
+        guard let rdv1 = centre1.prochainRdv?.toDate(region: region),
+              let rdv2 = centre2.prochainRdv?.toDate(region: region) else {
+            return false
+        }
+        return rdv1.isBeforeDate(rdv2, granularity: .second)
+    }
```

Comment:
Warning: this PR will need to be completed, as the distance filter currently does not work as expected.

Reply:
This will be completed with search by city feature, as the distance will be available when this feature will be too.